schemas/org/openstax/ec/types/source_uri.rb

```diff
@@ -0,0 +1,5 @@
+record :source_uri do
+  required :host, :string
```

Comment:
add protocol? e.g. app://

Reply:
ok - Ah that's right, Tom and I thought we'd skip it, you convinced me to add it back in. :)

Reply:
BTW, not happy w/ repeating the 'extends' in all the other types - should we be aggregating these into some base-type that is extended? 3 of the 4 types all extend the same 3 base-types (who_am_i, ordered_session, source_uri)  - started session extends session, instead of ordered_session.

Reply:
added (and refactored)